packaging/debs/Debian/debian/watch

```diff
@@ -1,4 +1,4 @@
-version=3
-
-https://www.rabbitmq.com/releases/rabbitmq-server/v(.*)/rabbitmq-server-(\d.*)\.tar\.gz \
-	debian uupdate
+version=4
+opts="filenamemangle=s%(?:.*?)?v?@PACKAGE@-(\d[\d.]*)\.tar\.xz%@PACKAGE@-$1.tar.xz%" \
+      https://github.com/rabbitmq/@PACKAGE@/releases \
+      (?:.*?/)?v?@PACKAGE@-(\d[\d.]*)\.tar\.xz debian uupdate
```

Comment:
Is this meant to be `update` or `uupdate`? I could not find an explanation in the Debian wiki. `uscan` works as expected but I thought I should clarify before merging.

Reply:
No, this is correct. The last argument specifies the script to run when upgrading the sources (https://manpages.debian.org/stretch/devscripts/uupdate.1.en.html) The  naming does look a little suspicious though :smile: